lib/Runtime/Library/JavascriptLibrary.cpp

```diff
@@ -2541,6 +2554,31 @@ namespace Js
 
 #undef INIT_ERROR
 
+    bool JavascriptLibrary::InitializeAggregateErrorConstructor(DynamicObject * constructor, DeferredTypeHandlerBase * typeHandler, DeferredInitializeMode mode)
```

Comment:
To make this neater please could you update the INIT_ERROR_IMPL and INIT_ERROR #define s above to include a length property then put AggregateError in with the others rather than re-writing it all?